test/cql-pytest/test_tablets.py

```diff
@@ -91,15 +91,22 @@ def test_tablets_can_be_explicitly_disabled(cql, skip_without_tablets):
         res = cql.execute(f"SELECT * FROM system_schema.scylla_keyspaces WHERE keyspace_name = '{keyspace}'")
         assert len(list(res)) == 0, "tablets replication strategy turned on"
 
-
-def test_alter_changes_initial_tablets(cql, skip_without_tablets):
+@pytest.mark.xfail(reason="issue #18801: Chaning initial tablets with ALTER keyspace needs `WITH replication = {`")
```

Comment:
https://github.com/scylladb/scylladb/issues/18801 was just fixed, so the test should pass now.